Source/WebCore/svg/SVGFilterPrimitiveStandardAttributes.cpp

```diff
@@ -53,18 +53,34 @@ void SVGFilterPrimitiveStandardAttributes::attributeChanged(const QualifiedName&
     SVGParsingError parseError = NoError;
 
     switch (name.nodeName()) {
-    case AttributeNames::xAttr:
-        Ref { m_x }->setBaseValInternal(SVGLengthValue::construct(SVGLengthMode::Width, newValue, parseError));
+    case AttributeNames::xAttr: {
+        auto length = SVGLengthValue::construct(SVGLengthMode::Width, newValue, parseError);
+        if (parseError != NoError || newValue.isNull())
+            length = SVGLengthValue(SVGLengthMode::Width, "0%"_s);
+        Ref { m_x }->setBaseValInternal(length);
```

Comment:
```suggestion
        parse(m_x, SVGLengthMode::Width, "0%"_s);
```
The `parse` function should be easy to write.

Reply:
Should the function be in SVGParserUtilities.cpp

Reply:
see https://github.com/WebKit/WebKit/pull/38414/files#r2106870367